SpringBoard/SBApplicationIcon.h

```diff
@@ -2,8 +2,16 @@
 
 @class SBApplication;
 
-@interface SBApplicationIcon : SBLeafIcon
+typedef struct SBIconImageInfo {
+	CGSize size;
+	CGFloat scale;
+	CGFloat continuousCornerRadius;
+} SBIconImageInfo;
```

Comment:
This struct exists in [SpringBoard/SBIconImageInfo.h](https://github.com/theos/headers/blob/819f49385304805f9a81e4210ae392c620505ea8/SpringBoard/SBIconImageInfo.h#L3-L7)